### PR TITLE
Fix #32 Doesn't work when the baseURL contains a path

### LINKED ISSUE #32: Doesn't work when the baseURL contains a path (BaseURLにパスが含まれるときにうまく動かないようです)
Thank you for the nice library.
In the enterprise version of Backlog, it doesn't seem to work when the BaseURL contains the path.
e.g. http://hoge.backlog.com/backlog/api/v2/issues

ナイスライブラリをありがとうございます。
エンタープライズ版Backlogで、BaseURLにパスが含まれるときにうまく動かないようです。
たとえば、http://hoge.backlog.com/backlog/api/v2/issues などのときです。

### PR DESCRIPTION
**Pull request by assout**

### CODE REVIEW

File: backlog_test.go [hunk 1]
```diff
@@ -17,10 +17,14 @@ var (
...
 
+func setup() (client *Client, mux *http.ServeMux, serverURL string, teardown func()) {
+	return setupWithPath("")
+}
+
 // setup sets up a test HTTP server along with a backlog.Client that is
 // configured to talk to that test server. Tests should register handlers on
 // mux which provide mock responses for the API method being tested.
-func setup() (client *Client, mux *http.ServeMux, serverURL string, teardown func()) {
+func setupWithPath(path string) (client *Client, mux *http.ServeMux, serverURL string, teardown func()) {
```
Comment (kenzo0107) [inline backlog_test.go:27]: I request to add comment about the method `setupWithPath(path string)` 🙏 

example:
```
// setup sets up a test HTTP server along with a backlog.Client that is
// configured to talk to that test server. Tests should register handlers on
// mux which provide mock responses for the API method being tested.
func setup() (client *Client, mux *http.ServeMux, serverURL string, teardown func()) {
	return setupWithPath("")
}

// setupWithPath sets up a test HTTP server along with a backlog.Client with the path.
func setupWithPath(path string) (client *Client, mux *http.ServeMux, serverURL string, teardown func()) {
...
```

**Review by kenzo0107 — COMMENTED** [review]
I commented! Please confirm it.

File: backlog_test.go (hunk 1 above)
Comment (assout) [reply backlog_test.go]: Fix comment.

**Review by kenzo0107 — APPROVED** [review]
LGTM 👍 

Thank you!

**Pull request merged**